GoldensMisc.cs

```diff
@@ -26,22 +26,27 @@ public class GoldensMisc : Mod
 	{
 		public static bool VanillaTweaksLoaded;
 		private static ReLogic.Content.Asset<Texture2D> CellPhoneTexture;
-		
+		private bool WormHoleHacksLoaded = false;
+		//0 is WormholeHacks
```

Comment:
Not sure what this comment means

Reply:
I was originally going to use a BitsByte to store a list of bools that monitored loading but I changed my mind and just used a  new bool for the only one that really needed it. I was going to add more comments to describe each bool but I felt it was reinventing the wheel so stopped. That comment is just left over sorry.

Reply:
Yeah I don't think this is a situation where you really need to save those 7 bytes of memory, good call